server/http_handler_test.go

```diff
@@ -739,15 +740,16 @@ func (ts *HTTPHandlerTestSuite) TestPostSettings(c *C) {
 	c.Assert(config.GetGlobalConfig().Log.Level, Equals, "error")
 	c.Assert(atomic.LoadUint32(&variable.ProcessGeneralLog), Equals, uint32(1))
 	form = make(url.Values)
-	form.Set("log_level", "info")
+	form.Set("log_level", "fatal")
```

Comment:
If ut in server failed, it will print a lot of info logs duration running `TestPostSettings` which is a bit noisy.